frontend/javascripts/oxalis/controller/combinations/bounding_box_handlers.ts

```diff
@@ -227,7 +227,8 @@ export function getClosestHoveredBoundingBox(
 export const highlightAndSetCursorOnHoveredBoundingBox = _.throttle(
   (delta: Point2, position: Point2, planeId: OrthoView) => {
     const hoveredEdgesInfo = getClosestHoveredBoundingBox(position, planeId);
-    const inputCatcher = document.getElementById(`inputcatcher_${planeId}`);
+    // Access the parent element as that is where the cursor style property is set
+    const inputCatcher = document.getElementById(`inputcatcher_${planeId}`)?.parentElement;
```

Comment:
During testing I noticed that the cursor never changed again after activating the bounding box tool, because the style property was set for the wrong element (and never was removed again).